docs/diagnostics/DiagnosticSuppression.md

```diff
@@ -0,0 +1,39 @@
+# Diagnostic Suppression
+If a diagnostic is not relevant to a specific code fragment and you have an explanation of why it is not relevant, you can suppress the diagnostic so that it is not displayed for the code fragment.
+
+You can suppress Acuminator diagnostics in one of the following ways (which are described in greater detail below):
+
+ - By adding a specific comment directly in code
+ - By adding a code fragment to the Acuminator suppression file
+
+## Diagnostic Suppression by Adding a Code Comment
+You can suppress any diagnostic by adding a specific comment directly in code. To add a suppression comment, you select **Suppress the PXYYYY diagnostic with Acuminator > in a comment** (where PXYYYY is the code of the diagnostic) as a code fix for the diagnostic, as shown in the following screenshot.
+
+![Suppression in a Comment](../images/SuppressDiagnosticInComment.png)
+
+The code below shows an example of a suppression comment that has been added by the code fix. A suppression comment includes the following parts:
+
+ - The code of the diagnostic (`PX1007` in the code example below).
+ - The name of the diagnostic (`NoXmlCommentForPublicClass` in the following code example), which can give a code reviewer an idea of what the suppressed diagnostic is about.
+ - The justification of the suppression, which explains why this diagnostic is suppressed for this code fragment. You type the explanation instead of `[Justification]`.
+
+```C#
+// Acuminator disable once PX1007 NoXmlCommentForPublicClass [Justification]
```

Comment:
If there are multiple diagnostics reported on the same line, then there might be multiple suppression comments one on top of each other. @k-s-popova I'm not sure if it's worth mentioning.